t/copy_input_fastq.t

```diff
@@ -0,0 +1,40 @@
+#!/bin/usr/env perl
+use warnings;
+use strict;
+
+use lib $FindBin::Bin.'../lib';
+use Test::More;
+use Test::Exception;
+use Stage::CopyInputFastq;
+use JobProperties;
+use Logger;
+use File::Temp 'tempdir';
+use Getopt::Long;
+
+#tests that CopyInputFastq.pm is correctly verifying that the input fastq and reference fasta have unique basenames.
+
+my $tmp_dir;
+my $keep_tmp;
+if (not GetOptions('t|tmp-dir=s' => \$tmp_dir,'k|keep-tmp' => \$keep_tmp))
+{
+        die "Error: No tmp-dir indicated.";
+}
+
+$keep_tmp = 0 if (not defined $keep_tmp);
+
+my $job_out = tempdir('copy_input_fastqXXXXXX', CLEANUP => (not $keep_tmp), DIR => $tmp_dir) or die "Could not create temp directory";
+my $logger = Logger->new("$job_out");
+my $properties = JobProperties->new('/Warehouse/Users/csieffert/core-phylogenomics/t');
```

Comment:
Could you pass the temporary directory here instead of the path on Warehouse?  These tests will fail if that directory does not exist.